bigquery_etl/util/udf_functions.py

```diff
@@ -19,7 +19,7 @@ def get_input(sql_text: str) -> Optional[str]:
 
 def get_output(sql_text: str) -> Optional[str]:
     """Extract output parameters from UDF sql code."""
-    cleaned_sql_text = re.sub(r"\s{2,}", " ", sql_text)
+    cleaned_sql_text = re.sub(r"\s{2,}|\n", " ", sql_text)
     output_groups = re.search(
         r"(?:RETURNS |CAST\([a-z,A-Z,0-9,_,.]+ AS |, OUT )(.+?)(?: AS|\s?\))",
```

Comment:
```suggestion
        r"(?:RETURNS |CAST\([a-z,A-Z,0-9,_,.]+ AS |, OUT )(.+?)(?: AS|\s?\)|\s?LANGUAGE)",
```
that way the edge case you found will give the correct output

Reply:
Awesome, committed this suggestion and updated the test, thanks!